windows/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM mcr.microsoft.com/windows/servercore:1809 AS caddy
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# https://github.com/caddyserver/dist/commits
+ENV CADDY_DIST_COMMIT 80870b227ded910971ecace4a0c136bf0ef46342
+
+RUN mkdir /caddy
+RUN mkdir /usr/share/caddy
+RUN Invoke-WebRequest \
+        -Uri "https://github.com/caddyserver/dist/raw/$env:CADDY_DIST_COMMIT/config/Caddyfile" \
+        -OutFile "/caddy/Caddyfile"
+RUN Invoke-WebRequest \
+        -Uri "https://github.com/caddyserver/dist/raw/$env:CADDY_DIST_COMMIT/welcome/index.html" \
+        -OutFile "/usr/share/caddy/index.html"
+
+# https://github.com/caddyserver/caddy/releases
+ENV CADDY_VERSION v2.0.0-rc.3
+
+RUN Invoke-WebRequest \
+        -Uri "https://github.com/caddyserver/caddy/releases/download/v2.0.0-rc.3/caddy_2.0.0-rc.3_windows_amd64.zip" \
+        -OutFile "/caddy.zip"; \
+    Expand-Archive -Path "/caddy.zip" -DestinationPath "/" -Force; \
+    Remove-Item "/caddy.zip" -Force
+
+# See https://caddyserver.com/docs/conventions#file-locations for details
+# ENV XDG_CONFIG_HOME=/config
+# ENV XDG_DATA_HOME=/data
+
+# VOLUME /config
+# VOLUME /data
```

Comment:
What do the volumes look like on Windows then?

Reply:
er... sorry, I committed but never pushed 😂 - this is out of date!